### PR TITLE
[CU-2w3b68x] docs(frame): extend vesting pallet readme

### PR DESCRIPTION
**Pull request by Martin-Kiss**
## Issue
https://app.clickup.com/t/2w3b68x

## Description
Extend vesting pallet readme, update existing content

## Checklist

- [x] Style, expression and spellchecking

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-2w3b68x Extend vesting pallet readme](https://app.clickup.com/t/2w3b68x)

File: frame/vesting/README.md [hunk 1]
```diff
@@ -1,17 +1,27 @@
-# Vesting Pallet
+# Vesting
+
+The vesting module provides a means for a scheduled balance lock on an account.
+It uses the graded vesting approach, which unlocks a specific amount of balance every period of time until all funds are unlocked.
```
Comment (benluelo) [inline frame/vesting/README.md:4]:
```suggestion
It utilizes the graded vesting approach, which unlocks a specific amount of balance every period of time until all funds are unlocked.
```

File: frame/vesting/README.md [hunk 2]
```diff
@@ -1,17 +1,27 @@
...
+- The original pallet is not currency agnostic. This fork is generalized to any currency and usable with the `MultiLockableCurrency` trait.
+- The pallet is modified to support measuring time windows in absolute timestamps and block numbers.
+
+## Vesting Schedule
 
-### Vesting Schedule
+The data structure `VestingSchedule` describes the schedule of a vesting plan:
+1. from the time of `window.start`,
+2. for every `window.period` amount of time,
+3. `per_period` an amount of balance would unlock, until
```
Comment (benluelo) [inline frame/vesting/README.md:17]:
```suggestion
3. `per_period` amount of balance is unlocked, until
```

File: frame/vesting/README.md [hunk 3]
```diff
@@ -1,17 +1,27 @@
...
+4. the number of periods 'period_count' is reached.  
 
-The schedule of a vesting is described by data structure `VestingSchedule`: from the time of `window.start`, for every `window.period` amount of time, `per_period` amount of balance would unlocked, until number of periods `period_count` reached. The pallet supports measuring time windows in terms of absolute timestamps as well as block numbers for vesting schedules. All `VestingSchedule`s under an account could be queried in chain state.
+Querying all `VestingSchedules` under an account could be done in chain state.
```
Comment (benluelo) [inline frame/vesting/README.md:20]:
```suggestion
All `VestingSchedules` under an account can be queried from the chain state.
```

File: frame/vesting/README.md [hunk 4]
```diff
@@ -1,17 +1,27 @@
...
 
-### Why fork
+## Workflows
 
-This tweaked version includes the following changes,
-1. The original Vesting pallet is not currency agnostic. This fork is generalized to any currency and usable with the `MultiLockableCurrency` trait. 
-2. Modified to support measuring time in terms of absolute timestamps as well as the original block number based scheme for vesting schedules.
+Initially, we create a `vested_transfer` to add a vesting schedule to an account. 
+Once created, a vesting schedule can be updated `update_vesting_schedules`.
```
Comment (benluelo) [inline frame/vesting/README.md:25]:
```suggestion
Once created, a vesting schedule can be updated with `update_vesting_schedules`.
```

File: frame/vesting/README.md [hunk 5]
```diff
@@ -1,17 +1,27 @@
...
 
-Other than that, most of the code is the original version from the [open runtime module repo](https://github.com/open-web3-stack/open-runtime-module-library/blob/1f520348f31b5e94b8a5dd7f8e6b8ec359df4177/vesting/README.md)
+We can `claim` any vested funds of an account directly to the call's origin account or `claim_for` to target a given account.
```
Comment (benluelo) [inline frame/vesting/README.md:27]:
```suggestion
Funds can be claimed in two ways; either directly with `claim` to claim for the caller, or indirectly through `claim_for` to claim for a given account.
```

**Comment by Martin-Kiss** [comment]
Please note pallet references to the products section will be done in a separate PR.

**Review by KaiserKarel — APPROVED** [review]

**Review by andor0 — APPROVED** [review]

**Comment by Martin-Kiss** [comment]
@Mergifyio refresh

**Pull request merged**